campaign/4e/char_power.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+  Please see the license.html file included with this distribution for 
+  attribution and copyright information.
+-->
+
+<root>
+    <windowclass name="char_power" merge="join" ruleset="4E">
```

Comment:
Code folding already struggles enough with FG XML, best to give it some assistance with consistent indenting.